README.md

```diff
@@ -1,6 +1,4 @@
-
-<a  href="https://www.freepnglogos.com/images/fortnite-png-27038.html" ><img  src="https://www.freepnglogos.com/uploads/fortnite-png/fortnite-logo-logo-fortnite-png-photo-icons-and-png-6.png"
-width="200" alt="fortnite logo, logo fortnite png photo icons and png" /></a>
+<a href="https://imgbb.com/"><img src="https://i.ibb.co/0Xkf3WD/logo.png" alt="logo" border="0"></a>
```

Comment:
@xroll we can use github links directly like this:

`<img src="https://github.com/AndreVarandas/dart-fortnite-api-wrapper/tree/main/art/logo.png" alt="logo" border="0">`

Would you do this last change? Thank you!

Reply:
Yes, understood !